### ISSUE TITLE
[Query] Request for WCAG 2.1 references for Accessibility Scanner Results

### ISSUE THREAD

**Issue by daphneyyt** [material-components/material-components-android#3543]
Hi Material Design team,

My team and I audit our clients' mobile apps for accessibility issues. Previously, we were using Accessibility Insights For Android. However, it is no longer in the market.

We are currently exploring using Accessibility Scanner for our future accessibility checks. One of our requirements is that our assessment must be based on WCAG 2.1.

We understand that Material's framework draws on WCAG standards and industry best practices to help anyone anticipate, plan, document, and implement accessible experiences. We would like to know if the MD team has a document reference that ties the Accessibility Scanner results to its corresponding WCAG 2.1 criteria?

Documents which we referenced:
[https://support.google.com/accessibility/android/answer/6376559?hl=en-GB&ref_topic=6376582&sjid=8528...](https://support.google.com/accessibility/android/answer/6376559?hl=en-GB&ref_topic=6376582&sjid=8528893810016795091-AP#zippy=%2Ccontent-labelling%2Cimplementation%2Ctouch-target-size%2Clow-contrast)

https://m3.material.io/foundations/accessible-design/design-to-implementation

Thank you
Daphne

**Comment by paulfthomas** [comment]
Hi Daphne,

Unfortunately we do not have this kind of mapping available. Also, afaik, the Accessibility Scanner for Android is not entirely based on WCAG which is primarily concerned with Web content. Sorry we cannot help more...
